sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -472,15 +472,19 @@ identifierComment
     ;
 
 relationPrimary
-    : tableIdentifier sample? (AS? strictIdentifier)?               #tableName
-    | '(' queryNoWith ')' sample? (AS? strictIdentifier)?           #aliasedQuery
-    | '(' relation ')' sample? (AS? strictIdentifier)?              #aliasedRelation
-    | inlineTable                                                   #inlineTableDefault2
-    | identifier '(' (expression (',' expression)*)? ')'            #tableValuedFunction
+    : tableIdentifier sample? (AS? strictIdentifier)?                 #tableName
+    | '(' queryNoWith ')' sample? (AS? strictIdentifier)?             #aliasedQuery
+    | '(' relation ')' sample? (AS? strictIdentifier)?                #aliasedRelation
+    | inlineTable                                                     #inlineTableDefault2
+    | identifier '(' (expression (',' expression)*)? ')' tableAlias   #tableValuedFunction
```

Comment:
Please follow [`inlineTable`](https://github.com/maropu/spark/blob/8d3a03712a4b3f55e60819cb8b6b3ec3454e12ab/sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4#L482-L484) to add a separate rule.